msbuild/Xamarin.MacDev.Tasks/Xamarin.MacDev.Tasks.csproj

```diff
@@ -78,6 +79,8 @@
     <Compile Include="Tasks\ReadItemsFromFile.cs" />
     <Compile Include="Tasks\Zip.cs" />
   </ItemGroup>
-  <ItemGroup />
+  <ItemGroup>
+    <None Include="packages.config" />
```

Comment:
We're not bringing another copy. And the project does not have the reference properly set up pointing to whichever version it's using from where place it has it. It's just referencing it from "wherever" and I guess it's being resolved to the Mono GAC on the Mac, but that magic just doesn't work on Windows ;). 

On the Mac and CI, we won't be bringing anything new at all. This is for the Windows/VS dev local builds (i.e. me! ;))